packages/ui/src/styles/helpers/zIndex.ts

```diff
@@ -0,0 +1,12 @@
+import { generate } from '../themes/utils';
+
+const vars = {
```

Comment:
It's not reasonable practice if this constants will not be changing dynamically. You should write them in constants way like in other helpers.